modules/custom/openy_activity_finder/templates/openy-activity-finder-program-search-page.html.twig

```diff
@@ -122,7 +122,6 @@
                 expander_sections_config = '{{ expanderSectionsConfig|json_encode }}'
                 :default='$route.query.days'
                 v-on:updated-values="days = $event"
-                :loading="loading"
```

Comment:
why loading removed?

Reply:
This option open **Days** filter each time when we wait for response from the controller. It lead to sidebar blinking.